client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx

```diff
@@ -79,7 +78,7 @@ const useIntroContent = ( flowName: string | null ): IntroContent => {
 		if ( isLinkInBioFlow( flowName ) ) {
 			return {
 				title: createInterpolateElement(
-					__( 'You’re 3 minutes away from<br />a stand-out Link in Bio site.<br />Ready? ' ),
+					__( 'You’re 3 minutes away from<br />a stand-out Link in Bio site.<br />Ready?' ),
```

Comment:
This change introduces a new translatable string, which concerned me. Looking at our "similar string" matching though, it should be copied as-is, so it's not a concern.

Reply:
Right, the editor decided to adjust them on save. Plz tell me if they should be reverted to their original state.

Reply:
I think they're fine. My initial thoughts were that we'd have to wait for translations, but our translation system is clever enough to cope with this 👍